Tests/Integration/IntegrationTestData.swift

```diff
@@ -104,7 +109,8 @@ let otpAppInfoValid = AppInfo(
     requiredIdentifier: "both",
     requireEmailVerification: false,
     requireIdentifierVerification: false,
-    sessionTimeoutLength: 6000
+    sessionTimeoutLength: 6000,
```

Comment:
Nitpick - could pull this out into a shared const since it's used in a few places. Non-blocking!

Reply:
Since its test data, we want the values explicitly set directly. For now at least. But good callout!